e2e-tests/docs/CONTRIBUTING.MD

```diff
@@ -0,0 +1,254 @@
+# Contributing
+
+We are excited that you want to be a part of this project by contributing to the end-to-end tests. Here is some information on how to get started, as well as our requirements and guidelines to ensure a smooth contribution process.
+
+## Getting Started
+
+### Clone and Install
+
+1. **Fork the Repository**
+
+   Click the "Fork" button at the top right corner of the repository page.
+
+2. **Clone Your Forked Repository**
+
+   ```bash
+   git clone https://github.com/your-username/backstage-showcase.git
+   ```
+
+3. **Navigate to the Project Directory**
+
+   ```bash
+   cd e2e-tests
+   ```
+
+4. **Install Dependencies**
+
+   We use **Yarn** for package management. If you don't have Yarn installed, you can install it globally:
+
+   ```bash
+   npm install -g yarn
+   ```
+
+   Then install the project dependencies:
+
+   ```bash
+   yarn install
+   ```
+
+5. **Install Playwright Browsers**
+
+   The Playwright browsers will be installed automatically via the `postinstall` script in `package.json`. If not, you can manually install them:
+
+   ```bash
+   yarn playwright install
+   ```
+
+6. **Run TypeScript Compilation**
+
+   Run type generation and checks:
+
+   ```bash
+   yarn tsc
+   ```
+
+### Running the Tests
+
+Our project contains multiple test suites for different environments and configurations. Some useful scripts to run the tests:
+
+```bash
+yarn showcase                       # Runs the showcase test suite
+yarn showcase-rbac                  # Runs the showcase RBAC test suite
+yarn showcase-1-2-x                 # Runs the showcase 1.2.x test suite
+yarn showcase-rbac-1-2-x            # Runs the showcase RBAC 1.2.x test suite
+yarn showcase-ci-nightly            # Runs the showcase CI nightly test suite
+yarn showcase-rbac-nightly          # Runs the showcase RBAC nightly test suite
+yarn showcase-aks-ci-nightly        # Runs the showcase AKS CI nightly test suite
+yarn showcase-rbac-aks-ci-nightly   # Runs the showcase RBAC AKS CI nightly test suite
+```
+
+### Useful Scripts
+
+Some additional scripts available in `package.json`:
+
+- **Linting:**
+
+  ```bash
+  yarn lint:check    # Checks for linting errors
+  yarn lint:fix      # Fixes linting errors automatically
+  ```
+
+- **Formatting:**
+
+  ```bash
+  yarn prettier:check   # Checks for formatting issues
+  yarn prettier:fix     # Fixes formatting issues automatically
+  ```
+
+## Contribution Guidelines
```

Comment:
I feel like some of this content might either duplicate or enhance what's in the contributing guide [here](https://github.com/janus-idp/backstage-showcase/blob/main/CONTRIBUTING.md) maybe some of this could be incorporated there and the more technical or detailed sections could go under `docs/e2e`.  Ideally I'd love to see e2e tests as part of a normal contribution as much as makes sense, you know?